config/betterquesting/DefaultQuests/Quests/AppliedEnergisti-AAAAAAAAAAAAAAAAAAAAIw==/AETier1FluidStor-AAAAAAAAAAAAAAAAAAAJrQ==.json

```diff
@@ -25,11 +25,11 @@
       "globalShare:1": 0,
       "questLogic:8": "AND",
       "repeat_relative:1": 1,
-      "name:8": "AE Fluid Storage",
+      "name:8": "AE Tier 1 Fluid Storage",
       "lockedProgress:1": 0,
       "autoClaim:1": 0,
       "isSilent:1": 0,
-      "desc:8": "The main processor used for 1K, 4K, 16K and 64K fluid components.\nAlso used to craft dual interfaces, as well as some other useful tools from AE2FC like the level maintainer.\n\nDon\u0027t want to get rid of the super tanks you have been using up to now? Use the fluid storage bus instead of the fluid storage cells from AE2FC then. Super tanks double in capacity each tier, but the cells quadruple. They\u0027re also less expensive for sure. But you decide which to use."
+      "desc:8": "The main Processor used for crafting 1K, 4K, 16K and 64K Fluid Storage Components. Aditionally its used to craft Dual Interfaces, as well as some other useful tools from AE2FC like the Level Maintainer.\n\nDon\u0027t want to get rid of the Super Tanks you have been using up to now? Use the Fluid Storage Bus instead of the Fluid Storage Cells from AE2FC then. Super Tanks double in capacity each tier, but the Cells quadruple. They\u0027re also less expensive for sure. But you decide which to use."
```

Comment:
_its_ typo

Reply:
My weary eyes missed those but all typos should be fixed now